control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1544,6 +1544,40 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/volumes/{volume_id}/properties/{property_name}/{property_value}':
```

Comment:
`properties` --> `property` because we are changing only one?

Also should it be in body rather than path? Let's say in future we want to change multiple properties? @tiagolobocastro wdyt?

Reply:
hmm actually I think plurality is correct, think of: /volumes/{volume_id} <- single volume as well right?
For multiple values we can use put `/volumes/{volume_id}/properties/{property_name}` with body perhaps?